server/api/cvrs.py

```diff
@@ -1041,7 +1042,7 @@ def parse_cvr_ballots() -> Iterable[CvrBallot]:
             batch_sequence = find(cvr_xml, "BatchSequence").text
 
             db_batch = batches_by_key.get(
-                (cvr_zip_file_name.strip(".zip"), batch_number)
+                (cvr_zip_file_name_without_extension, batch_number)
```

Comment:
`.strip(".zip")` actually tells Python to remove any consecutive sequence (order doesn't matter) of `.`, `z`, `i`, or `p` from the start and end of the string. So `"in-person.zip".strip(".zip")` actually returns `"n-person"` 🤦. Definitely a fun find 😅